rust/main/utils/run-locally/src/cosmos/mod.rs

```diff
@@ -94,58 +94,25 @@ pub struct MockDispatchInner {
     pub metadata: String,
 }
```

Comment:
would rather leave the files in `run-locally/src/cosmos` unchanged, are these changes intended?

Reply:
yes this code was literally not up to date

Reply:
could probably be fixed in another pr tho

Reply:
It should be as easy as overwriting these files with what's on the `main` branch right? Let's undo any `run-locally/src/cosmos` from this PR to keep it starknet specific

Reply:
done